python/GafferSceneUI/SceneInspector.py

```diff
@@ -727,6 +727,12 @@ def supportsInheritance( self ) :
 
 		return False
 
+	## Should return True if the Inspector's labels
+	# contain values appropriate for dragging.
+	def supportsLabelDrag( self ) :
+
+		return True
```

Comment:
I wonder if this distinction could be overly fussy? I don't see much harm in dragging a label like "Local Transform", and it might be easier for folks to find the feature if it applies consistently to all labels. With my refactoring-to-use-the-new-inspector-framework hat on, it'd also be nice to have one less bit of fiddliness to port over.

Reply:
Overly fussy? Likely so! :laughing:  At the time it felt appropriate to differentiate the two into labels that contain useful data and labels that are just labels, but I've dropped that in the latest push as there's no harm in someone dragging "Local Transform", it does make the drags more consistent, and will for sure save us some future effort in the port to the new inspectors...